update decex workflow to include Sudo operations on domain (#44)
autonomys/protocol-specs: docs/decex/workflow.md

This inherent extrinsic also affects the `FraudProof::InvalidDomainExtrinsicRoot` if any malicious operator does not include this inherent while importing Domain block.
Honest operators will submit above FraudProof variant with all the necessary storage proofs to construct the Domain Extrinsic root.

## Domain Freeze, Unfreeze, and Prune Execution Receipt by Consensus Sudo
Generally, malicious activity from a domain operator is handled through Fraud proofs where honest operators verify the bundles before importing domain block and submit

> Fraud proof targeted at given bad Execution receipt. In a particular case where Fraud proof would not fit into the Consensus block due to weight or size restrictions of the

Reviewer: ```suggestion
Fraud proof targeted at given bad Execution receipt. In a particular case where Fraud proof could not be verified automatically or included in the
```
wanted this to be more generic than just size/weight
